addon/helpers/can-access-route.ts

```diff
@@ -1,53 +1,60 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
 import { EVENTS } from '@bagaar/ember-permissions/-private/config';
+import PermissionsService from '@bagaar/ember-permissions/services/permissions';
 import Helper from '@ember/component/helper';
 import { action } from '@ember/object';
 import { addListener, removeListener } from '@ember/object/events';
 import { inject as service } from '@ember/service';
 
 export default class CanAccessRouteHelper extends Helper {
-  @service('permissions') permissionsService;
+  @service('permissions') declare permissionsService: PermissionsService;
 
-  constructor() {
-    super(...arguments);
+  constructor(owner: unknown) {
+    // @ts-ignore
+    super(owner);
```

Comment:
The DefinitelyTyped(DT) type is wrong. It expects a `properties: {}` instead of an owner.

I'm not sure in which version this was released, but al lot of framework classes have been standardized and receive the owner as the first argument in their constructor: https://github.com/emberjs/rfcs/pull/451

Should I mention this somewhere? I plan on opening an issue in DT for this but navigating that repo is a pain :smile:.